app/design/frontend/Magento/luma/Magento_Theme/web/css/source/_module.less

```diff
@@ -525,6 +525,9 @@
 
         .customer-name {
             cursor: pointer;
+            span {
+                display: none;
```

Comment:
Why do we need to hide it? Simply eliminate redundant text rendering

Reply:
made the changes for above